### PR TITLE
TW review of mithril-client.md

### PR DESCRIPTION
**Pull request by olgahryniuk**

### CODE REVIEW

File: docs/root/manual/developer-docs/nodes/mithril-client.md [hunk 1]
```diff
@@ -231,28 +231,28 @@ mithril_client help
...
 
-# 3- Show detailed informations about a snapshot
+# 3- Show detailed information about a snapshot
 mithril_client snapshot show $SNAPSHOT_DIGEST
 
 # 4- Download the given snapshot and verify the certificate
 mithril_client snapshot download $SNAPSHOT_DIGEST
 
-# 5- List Mithril Stake Distributions
+# 5- List Mithril stake distribution
```
Comment (jpraynaud) [inline docs/root/manual/developer-docs/nodes/mithril-client.md:240]:
```suggestion
# 5- List Mithril stake distributions
```

File: docs/root/manual/developer-docs/nodes/mithril-client.md [hunk 2]
```diff
@@ -266,33 +266,32 @@ Here are the subcommands available:
...
+### Mithril stake distribution
 
 | Subcommand | Performed action |
 |------------|------------------|
-| **download** | Download and verify a Mithril Stake Distribution|
-| **help** | Print this message or the help of the given subcommand(s)|
-| **list** | List available Mithril Stake Distributions|
+| **download** | Downloads and verifies Mithril stake distribution|
+| **help** | Prints this message or the help for the given subcommand(s)|
+| **list** | Lists available Mithril stake distribution|
```
Comment (jpraynaud) [inline docs/root/manual/developer-docs/nodes/mithril-client.md:280]:
```suggestion
| **list** | Lists available Mithril stake distributions|
```

**Review by jpraynaud — CHANGES_REQUESTED** [review]
I noticed 2 typos, as we have multiple stake distributions that are listed :slightly_smiling_face:

**Comment by olgahryniuk** [comment]
Thanks @jpraynaud ! Noted for the future

**Review by jpraynaud — APPROVED** [review]
LGTM :+1:

**Pull request merged**